subsys/bootloader/Kconfig

```diff
@@ -154,7 +154,7 @@ config SB_BPROT_IN_DEBUG
 endif # SECURE_BOOT_DEBUG
 endif # IS_SECURE_BOOTLOADER
 
-if IS_SECURE_BOOTLOADER || SECURE_BOOT
+if IS_SECURE_BOOTLOADER || SECURE_BOOT || SECURE_BOOT_CRYPTO_CLIENT
```

Comment:
Maybe not the best Kconfig option but images which uses the `SECURE_BOOT_CRYPTO_CLIENT` option needs the configurations given by the following Kconfigs